src/containers/shared/components/Streams.jsx

```diff
@@ -68,9 +68,9 @@ const fetchLoadFee = (rippledSocket) =>
 
 const formatLedgers = (data) =>
   Object.entries(data)
-    .map((l) => {
-      const hashes = Object.entries(l[1].hashes || {}).map((h) => {
-        const validated = h[0] === l[1].ledger_hash
+    .map(([_index, ledger]) => {
```

Comment:
Was this change due to _index not being used?

Reply:
Yes